ceph.spec.in

```diff
@@ -537,6 +537,7 @@ Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-jwt
 Requires:       python%{python3_pkgversion}-routes
 Requires:       python%{python3_pkgversion}-werkzeug
+Suggests:       python%{python3_pkgversion}-saml
```

Comment:
please use something like
```
%if 0%{?weak_deps}
Recommends:    python%{python3_pkgversion}-saml
%endif
```

we can drop the guards if we decide to drop the el7 support. but let's do this in a different PR if we really want to do this.

Reply:
Done!